src/ProjectFileManager.cpp

```diff
@@ -16,53 +16,58 @@ Paul Licameli split from AudacityProject.cpp
 #include <wx/evtloop.h>
 #endif
 
-#include <wx/frame.h>
-#include <wx/log.h>
+#include "AdornedRulerPanel.h"
+#include "AudacityMessageBox.h"
+#include "AudacityDontAskAgainMessageDialog.h"
 #include "BasicUI.h"
 #include "CodeConversions.h"
+#include "Export.h"
+#include "HelpText.h"
+#include "Import.h"
+#include "ImportPlugin.h"
+#include "ImportProgressListener.h"
 #include "Legacy.h"
+#include "MusicInformationRetrieval.h"
 #include "PlatformCompatibility.h"
 #include "Project.h"
 #include "ProjectFileIO.h"
 #include "ProjectHistory.h"
 #include "ProjectNumericFormats.h"
-#include "ProjectSelectionManager.h"
-#include "ProjectWindows.h"
 #include "ProjectRate.h"
+#include "ProjectSelectionManager.h"
 #include "ProjectSettings.h"
 #include "ProjectStatus.h"
 #include "ProjectTimeSignature.h"
 #include "ProjectWindow.h"
+#include "ProjectWindows.h"
+#include "RealtimeEffectList.h"
 #include "SelectFile.h"
 #include "SelectUtilities.h"
 #include "SelectionState.h"
 #include "Tags.h"
 #include "TempDirectory.h"
+#include "TimeDisplayMode.h"
 #include "TrackFocus.h"
 #include "TrackPanel.h"
+#include "TrackPanelAx.h"
 #include "UndoManager.h"
+#include "WaveClip.h"
 #include "WaveTrack.h"
-#include "wxFileNameWrapper.h"
-#include "Export.h"
-#include "Import.h"
-#include "ImportProgressListener.h"
-#include "ImportPlugin.h"
+#include "XMLFileReader.h"
 #include "import/ImportMIDI.h"
 #include "import/ImportStreamDialog.h"
+#include "prefs/MusicInformationRetrievalPrefs.h"
 #include "toolbars/SelectionBar.h"
-#include "AudacityMessageBox.h"
+#include "tracks/playabletrack/wavetrack/WaveTrackUtils.h"
 #include "widgets/FileHistory.h"
 #include "widgets/UnwritableLocationErrorDialog.h"
 #include "widgets/Warning.h"
+#include "wxFileNameWrapper.h"
```

Comment:
And `#include` reorderings...

Reply:
Obviously devs tried to manually keep these in order, and yet it was imperfect. Automatic re-ordering is painful for the reviewers when not done regularly, sorry about that, but then everyone could do it automatically and time would be spared.